### PR TITLE
Update README.textile

### PR DESCRIPTION
**Pull request by Dinduks**

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -20,9 +20,9 @@ pre. sudo python setup.py install
 
 h5. Usage
 
-Drag a selection in the grid and then click a window to automatically resize it to fill that portion of the screen.
+Show/hide the grid by pressing <code><Ctrl>+<Alt>+D</code> (currently not configurable).
 
-Show/hide the grid by pressing <Alt>R (currently not configurable).
+Drag a selection in the grid and then click a window to automatically resize it to fill that portion of the screen.
 
 Exit the application by right-clicking on its icon in the status bar and selecting Quit.
```
Comment (Dinduks) [inline README.textile]: Is there an icon for Ration?

**Comment by joaquincasares** [comment]
Hey @Dinduks ,

I just made some changes that broke merging this pull request. Could you update your branch and submit another?

Thanks!

File: README.textile (hunk 1 above)
Comment (joaquincasares) [inline README.textile]: I see a little square with 4 inner squares when using ration with xubuntu. Not sure where that's set in the repo.

**Comment by Dinduks** [comment]
I see no icon here (Pantheon, elementary OS).

**Comment by joaquincasares** [comment]
Nice, been wanting to give Pantheon a go.

I have no plans to look into icon support since I'm only using it for my personal computer and onyxfish has moved on as well, but feel free to submit a pull request for adding an icon. This would be nice to have.

**Pull request closed without merging**